codeworld-api/src/CodeWorld/Color.hs

```diff
@@ -93,6 +97,30 @@ gray, grey :: Double -> Color
 gray = grey
 grey k = RGBA k k k 1
 
+
+-- Produce an infinite list of colors:
+-- https://github.com/google/codeworld/issues/436
+-- The list loops through the HSL spectrum with fixed
+-- saturation and lightness:
+-- https://www.w3schools.com/colors/colors_hsl.asp
+-- http://stackoverflow.com/a/7728626/1007926
+
+colorHues :: [Double]
+colorHues = let
```

Comment:
This looks great, thanks!

I think we can make one of the changes I suggested pretty easily: starting with the primary and tertiary colors.  That would look something like this:

    colors :: [Color]
    colors = red : green : blue : unfoldr nextColor (1, 6)
      where nextColor :: (Int, Int) -> Maybe (Double, (Int, Int))
            nextColor (numer, denom) =
                let hue = fromIntegral numer * 2 * pi / fromIntegral denom
                    color = fromHSL hue 0.75 0.5
                    numer' = (numer + 2) `mod` denom
                    denom' = if numer' == 1 then 2 * denom else denom
                in  Just (color, (numer', denom'))

Thoughts?